WalletWasabi.Fluent/Converters/IndentConverter.cs

```diff
@@ -0,0 +1,31 @@
+using System.Globalization;
+using Avalonia;
+using Avalonia.Data.Converters;
+
+namespace WalletWasabi.Fluent.Converters;
+
+public class IndentConverter : AvaloniaObject, IValueConverter
+{
+	public static readonly StyledProperty<double> MultiplierProperty = AvaloniaProperty.Register<IndentConverter, double>(nameof(Multiplier));
```

Comment:
Do we plan do bindings to Multiplier property? If not this might be just common clr property.